jf_agent/jf_jira/jira_download.py

```diff
@@ -530,7 +536,10 @@ def _strip_changelog(c):
         # copy a changelog, stripping excluded fields from the history
         return {**c, 'histories': list(_strip_changelog_histories(c['histories']))}
 
-    return [{**i, 'changelog': _strip_changelog(i['changelog'])} for i in issues]
+    if not include_fields and not exclude_fields:
+        return issues
+    else:
+        return [{**i, 'changelog': _strip_changelog(i['changelog'])} for i in issues]
```

Comment:
Correct @eliwind not strictly necessary but I think a small improvement in efficiency and unexpected data issues down the road...